src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountPrimitives.fs

```diff
@@ -0,0 +1,39 @@
+﻿module Deciders.BankAccountPrimitives
+
+open System
+
+[<Struct>]
+type AccountId = AccountId of Guid
+
+module AccountId =
+    let newId () = AccountId(Guid.NewGuid())
+
+[<Struct>]
+type ClientId = ClientId of Guid
+
+module ClientId =
+    let newId () = ClientId(Guid.NewGuid())
+
+[<Struct>]
+type AtmId = AtmId of Guid
+
+module AtmId =
+    let newId () = AtmId(Guid.NewGuid())
+
+[<Struct>]
+type CashierId = CashierId of Guid
+
+module CashierId =
+    let newId () = CashierId(Guid.NewGuid())
+
+[<Struct>]
+type CurrencyCode = CurrencyCode of string
+
+module CurrencyCode =
+    let newCode code = CurrencyCode(code)
```

Comment:
I find `parse` and `toString` work well as the names of the helpers to map to/from raw values

Reply:
Parens around `code` not required.

Reply:
Updated in https://github.com/oskardudycz/Ogooreck/pull/13/commits/db717592801bbff360e02f0b7e5bd6a3e5d9c1a0.